app/assets/javascripts/calendars/guiders-appointments.es6

```diff
@@ -57,7 +60,13 @@
     }
 
     getHeight() {
-      return $(window).height() - this.$el.offset().top - $('.page-footer').outerHeight(true);
+      let height = $(window).height() - this.$el.offset().top - $('.page-footer').outerHeight(true);
+
+      if (this.actionPanel.length && this.actionPanel.is(':visible')) {
```

Comment:
is `this.actionPanel` always a jQuery obj? If so I think just `this.actionPanel.is(':visible')` would cover this check?